### DISCUSSION TITLE
【OpenGL (ES)】描画結果を取得し、BMPとして保存する - わたすけのへや

### DISCUSSION THREAD

**Discussion by giscus** [watasuke102/watasuke.net#1] [Article]
# 【OpenGL (ES)】描画結果を取得し、BMPとして保存する - わたすけのへや

OpenGLあるいはOpenGL ESで、Framebufferへ描画した結果をBMPファイルとして保存する方法をまとめました。

https://watasuke.net/blog/article/save-opengl-result-as-bmp/

**Comment by yuto-trd** [comment]
Pixel Buffer Objectを使うと負荷が軽くなるみたいですよ🤓

**Reply by watasuke102** [reply]
知りませんでした、ありがとうございます！PBOを使った実装＋実行時間の比較について追記しておきました：

https://watasuke.net/blog/article/save-opengl-result-as-bmp/#pboを用いた高速化2024-11-20追記
